src/world.js

```diff
@@ -27,6 +28,11 @@ class World extends EventEmitter {
     if (storageProvider && savingInterval !== 0) this.startSaving()
   }
 
+  setRegistry (registry) {
```

Comment:
This is a bit weird, since prismarine-world needs an instance of prismarine-registry in order to build the block cache. There is a loader function in prismarine-world, but because of the async/sync setup inside prismarine-world, it's not possible to wrap the class inside the loader function.

A breaking refactor might be needed to fix the async/sync situation